ui/seek_bar.js

```diff
@@ -799,32 +802,25 @@ shaka.ui.SeekBar = class extends shaka.ui.RangeElement {
           c.el.classList.toggle(hiddenClass);
         }
       }
-    }, chapterEventOptions);
+    }, {passive: true});
 
-    this.eventManager.listen(this.bar, 'pointerout', () => {
+    this.chaptersEventManager_.listen(this.bar, 'pointerout', () => {
       for (const c of chapterElMap) {
         if (!c.el.classList.contains(hiddenClass)) {
           c.el.classList.add(hiddenClass);
         }
       }
-    }, chapterEventOptions);
-
-    container.insertBefore(
-        chaptersContainer, this.container.childNodes[0]);
+    }, {passive: true});
   }
 
   /**
-   * @param {Element} parent
    * @private
    */
-  recursivelyRemove_(parent) {
-    if (!parent) {
-      return;
-    }
-    while (parent.lastChild) {
-      this.recursivelyRemove_(parent.lastElementChild);
+  deletePreviousChapters_() {
+    this.chaptersEventManager_.removeAll();
+    while (this.chaptersContainer_.lastChild) {
```

Comment:
You could also use `shaka.util.Dom.removeAllChildren` here.

Reply:
Changed!